specification/sdk-tracing.md

```diff
@@ -260,6 +260,40 @@ high contention in a very high traffic service.
 * `maxExportBatchSize` - the maximum batch size of every export. It must be
   smaller or equal to `maxQueueSize`. The default value is `512`.
 
+##### Span watcher processor
+
+This implementation passes spans that have been started but not yet ended
+to the configured exporter.
+On each interval defined by `reportIntervalMillis`, all unfinished spans that
```

Comment:
Is "unfinished" a well-defined term? I always use "un-ended" because even though it does sound a bit awkward, it is at least very clear.

Reply:
You're right, "finished" could be interpreted as "done with exporting" as well, for example. I'll change it to "un-ended" until we find a less awkward term.